docker-compose.yml

```diff
@@ -23,13 +27,18 @@ services:
             - XMPP_BOSH_URL_BASE=http://xmpp.meet.jitsi:5280
             - XMPP_GUEST_DOMAIN
             - XMPP_MUC_DOMAIN
+            - XMPP_RECORDER_DOMAIN
             - TZ
         networks:
             meet.jitsi:
+                aliases:
+                    - meet.jitsi
 
     # XMPP server
     prosody:
         image: jitsi/prosody
+        ports:
```

Comment:
why is this needed?

Reply:
it needs for connect to prosody from external jibri instance (case 2)

Reply:
Then drop it, let's start with case 1.